diff --git LICENSE LICENSE
old mode 100644
new mode 100755
diff --git README.md README.md
old mode 100644
new mode 100755
diff --git src/facebox.js src/facebox.js
index 7d0e39d..268c455 100644
--- src/facebox.js
+++ src/facebox.js
@@ -84,13 +84,13 @@
     settings: {
       opacity      : 0.2,
       overlay      : true,
-      loadingImage : '/facebox/loading.gif',
-      closeImage   : '/facebox/closelabel.png',
+      loadingImage : '/site_media/static/images/facebox/loading.gif',
+      closeImage   : '/site_media/static/images/facebox/closelabel.png',
       imageTypes   : [ 'png', 'jpg', 'jpeg', 'gif' ],
       faceboxHtml  : '\
     <div id="facebox" style="display:none;"> \
       <div class="popup"> \
-        <div class="content"> \
+        <div class="content" style="width:500px;"> \
         </div> \
         <a href="#" class="close"></a> \
       </div> \
