*** urls.py.orig	2011-05-23 22:08:33.000000000 -0400
--- urls.py	2011-05-23 22:07:44.000000000 -0400
***************
*** 5,10 ****
  urlpatterns = patterns("",
      url(r"^username_autocomplete/$", "pinax.apps.autocomplete_app.views.username_autocomplete_friends", name="profile_username_autocomplete"),
      url(r"^$", "pinax.apps.profiles.views.profiles", name="profile_list"),
!     url(r"^profile/(?P<username>[\w\._-]+)/$", "pinax.apps.profiles.views.profile", name="profile_detail"),
      url(r"^edit/$", "pinax.apps.profiles.views.profile_edit", name="profile_edit"),
  )
--- 5,10 ----
  urlpatterns = patterns("",
      url(r"^username_autocomplete/$", "pinax.apps.autocomplete_app.views.username_autocomplete_friends", name="profile_username_autocomplete"),
      url(r"^$", "pinax.apps.profiles.views.profiles", name="profile_list"),
!     url(r"^profile/(?P<username>[@\w\._-]+)/$", "pinax.apps.profiles.views.profile", name="profile_detail"),
      url(r"^edit/$", "pinax.apps.profiles.views.profile_edit", name="profile_edit"),
  )
